spec/camelcase.js

```diff
@@ -0,0 +1,6 @@
+module.exports =function ( str ) {
+  function upperCase(match) {
+    return match.toUpperCase()
+  }
+  return str.replace(/\b[a-z]/g, upperCase).replace(/\s/g, '')
+}
```

Comment:
this is good, but I would challenge you to try and not use built in js libraries to do this...so you can get more experience problem solving